oneview/resource_hypervisor_cluster_profile.go

```diff
@@ -452,11 +498,13 @@ func resourceHypervisorClusterProfile() *schema.Resource {
 			"status": {
 				Type:     schema.TypeString,
 				Optional: true,
+				Computed: true,
```

Comment:
i think it should be set as computed: true no optional

Reply:
This also not a read only attribute

Reply:
state, status, type, etag, created, modified these are common properties for any resource calls. By nature those are read only